requirements.txt

```
@@ -1 +1,2 @@
-jupyterhub>=0.5
\ No newline at end of file
+jupyterhub>=0.5
+pyjwt==1.7.1
```

We should not pin on the latest release only, what about pyjwt>=1.7.1 ?

```suggestion
pyjwt>=1.7.1
```

---

Also, remove pyjwt from test-requirements.txt is needed if this is added in requirements.txt

---

I don't see why it could not be changed. During the project we were pinning most of the packages to guarantee some reproducibility, but we're not using some very specific function here, just decoding the JWT.

---

It is fine to pin packages, but we should not make oauthenticator as a package require specifically 1.7.1, then you wont be able to install both oauthenticator and a package that require a version of 1.7.2 or later.